.github/workflows/xtest.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v2
```

Comment:
Should we add setup-python as well? This will guarantee we have the same (major at least) version of python from run to run, and might save a few seconds by caching the opentdf library

https://github.com/actions/setup-python

Reply:
added!